lib/src/widgets/dashboard_screen/card_maturity_donut_chart.dart

```diff
@@ -13,15 +13,16 @@ class CardMaturityDonutChart extends StatelessWidget {
   Widget build(BuildContext context) {
     return new charts.PieChart(seriesList,
         animate: animate,
-        defaultRenderer: new charts.ArcRendererConfig(
-            arcWidth: 10,
-            arcRendererDecorators: [new charts.ArcLabelDecorator()]));
+        defaultRenderer: new charts.ArcRendererConfig(arcWidth: 4));
   }
 }
 
 class MatureCount {
   final String title;
   final int maturity;
+  final charts.Color color;
 
-  MatureCount(this.title, this.maturity);
+  MatureCount(this.title, this.maturity, Color color)
+      : this.color = new charts.Color(
+            r: color.red, g: color.green, b: color.blue, a: color.alpha);
```

Comment:
Could you put a comma at the end of arguments and format (just saving the file) it?

Reply:
Done